conda_smithy/configure_feedstock.py

```diff
@@ -1195,9 +1195,29 @@ def render_README(jinja_env, forge_config, forge_dir):
             fh.write(line)
 
 
+def generate_gitignore(forge_dir):
+    default_lines = ["*.pyc", "build_artifacts"]
```

Comment:
These default lines should be part of the forge_config. I think it makes the most sense for people to need to change the `conda-forge.yml`  file for the gitignore file to be updated, in the same way that all configurations are stored for all other feedstock files that conda-smithy manages.  This implementation seems to avoid that.